storage-node/src/server.rs

```diff
@@ -59,8 +60,14 @@ mod tests {
     use reqwest::Client;
     use std::fs;
     use std::io::Write;
+    use storage_common::init_logger;
     use tempfile::tempdir;
 
+    #[test]
+    fn setup() {
+        init_logger();
+    }
```

Comment:
Is there a way not to do this in every `mod tests`? For example if the log is not initialized we just do nothing for logging macros?

Reply:
> For example if the log is not initialized we just do nothing for logging macros?

wdym?

Reply:
The contents of the log should only be useful in non-test environments. When we're doing unit tests we actually don't need the log output. But it seems redundant to write this every time we write unit tests for a new struct. So is it possible that we don't do this but the code can still compile and the ut can still run?